modules/021-cni-cilium/images/cilium/Dockerfile

```diff
@@ -59,34 +60,26 @@ LABEL maintainer="maintainer@cilium.io"
 COPY --from=rootfs / /
 
 
+#
 # cilium-envoy from github.com/cilium/proxy
 #
-FROM quay.io/cilium/cilium-envoy:9c0d933166ba192713f9e2fc3901f788557286ee@sha256:943f1f522bdfcb1ca3fe951bd8186c41b970afa254096513ae6e0e0efda1a10d as cilium-envoy
+# Image source: https://github.com/cilium/cilium/blob/v1.12.4/images/cilium/Dockerfile#L11
+FROM quay.io/cilium/cilium-envoy:48cbb2f88edca93ba80d1135dbb067a827b8adc2@sha256:2c21f4a37bf6d46da88758a6194c0e63ab26c80a91c41d679247a921c46d42d3 as cilium-envoy
 
 #
 # Hubble CLI
 #
 FROM ${CILIUM_BUILDER_IMAGE} as hubble
-RUN mkdir /tmp/cilium-repo && curl -sSL https://github.com/cilium/cilium/archive/refs/tags/v1.11.5.tar.gz | tar xvz -C /tmp/cilium-repo
-RUN bash /tmp/cilium-repo/cilium-1.11.5/images/cilium/download-hubble.sh
+RUN mkdir /tmp/cilium-repo && curl -sSL https://github.com/cilium/cilium/archive/refs/tags/v1.12.4.tar.gz | tar xvz -C /tmp/cilium-repo
```

Comment:
Do we skip dropping additional RBAC or would we do this in another PR?

Reply:
which RBAC?

Reply:
https://github.com/deckhouse/deckhouse/issues/2961